addons/web/static/tests/legacy/fields/relational_fields_tests.js

```diff
@@ -1019,6 +1030,47 @@ QUnit.module('relational_fields', {
         form.destroy();
     });
 
+    QUnit.test('statusbar edited by the smart action "Move to stage..."', async function (assert) {
+        assert.expect(3);
+
+        const legacyEnv = makeTestEnvironment({ bus: core.bus });
+        const serviceRegistry = registry.category("services");
+        serviceRegistry.add("legacy_command", makeLegacyCommandService(legacyEnv));
+
+        const views = {
+            'partner,false,form': '<form>' +
+                     '<field name="stage" widget="statusbar" options=\'{"clickable": "1"}\' />' +
+                  '</form>',
+            'partner,false,search': '<search></search>',
+        };
+        const serverData = { models: this.data, views}
+        const webClient = await createWebClient({serverData});
```

Comment:
no need to create a webclient for this test, you should spawn a form view instead. If you did this because you need the wowl env to be there for the hotkey service, at least add a comment on the top of the test to tell us that we can replace webclient by formview when we'll convert it to wowl.

Reply:
@aab-odoo , Added the comment to convert it to wowl. Thanks